ocaml/idl/datamodel_cluster.ml

```diff
@@ -194,6 +194,16 @@ let t =
              (Some (VString Constants.default_smapiv3_cluster_stack))
            "Simply the string 'corosync'. No other cluster stacks are \
             currently supported"
+       ; field ~qualifier:DynamicRO ~lifecycle:[] ~ty:Bool "is_quorate"
+           ~default_value:(Some (VBool false)) "Whether the cluster is quorate"
+       ; field ~qualifier:DynamicRO ~lifecycle:[] ~ty:(Set String)
+           "quorum_members" ~default_value:(Some (VSet []))
+           "Current members that are in the quorum"
+       ; field ~qualifier:DynamicRO ~lifecycle:[] ~ty:Int "quorum"
+           ~default_value:(Some (VInt 0L))
+           "Number of nodes in order to be quorate"
+       ; field ~qualifier:DynamicRO ~lifecycle:[] ~ty:Int "total_votes"
```

Comment:
Is this simply the number of elements in `quorum_members`?

Reply:
I think the semantics of the vote is up to the corosync implementation. There are potential complicatiosn where quorum-device/pseudo node can have votes and nodes with more than one vote (very rare). But it does seem to me that in most cases this would be the same as the length of `quorum_members`